app/validation/error_messages.py

```diff
@@ -11,6 +11,9 @@
     'NUMBER_TOO_LARGE': 'Enter an answer less than or equal to %(max)s.',
     'NUMBER_TOO_SMALL_EXCLUSIVE': 'Enter an answer more than %(min)s.',
     'NUMBER_TOO_LARGE_EXCLUSIVE': 'Enter an answer less than %(max)s.',
+    'SUM_NOT_EQUAL_TO_TOTAL': 'Enter answers that add up to the %(label)s, %(total)s.',
+    'SUM_NOT_EQUAL_TO_OR_LESS_THAN_TOTAL': 'Enter answers that add up to or are less than the %(label)s, %(total)s.',
+    'SUM_NOT_LESS_THAN_TOTAL': 'Enter answers that are less than the %(label)s, %(total)s.',
```

Comment:
Can you also display the calculated total for the user's entry?. I think it would be useful to show what the total should be as this saves them having to go back to the previous page to check. This can be very handy for the averages questions.

Also, I don't think using words like `"add up to"` in the error message is correct for the average questions. It could easily be misinterpreted. Best to have something like `"Enter answers that average to"` etc. Because my numbers add up to ie 7 but doesn't average to 7.

Reply:
So average right now is just a feature, and wont be used until its wanted, at which point new error messages should be implemented also

Reply:
`'Enter answers that add up to the %(label)s, %(total)s.'` uses old style formatting, have you considered something like:

`'Enter answers that add up to the {label}, {total}.'`
then something like
```
self.messages['SUM_NOT_EQUAL_TO_OR_LESS_THAN_TOTAL'].format(**{'label': foo, 'total': bar})
```
more readable/neater?

Reply:
Agree with this re. formatting. We have a refactoring ticket for this whole area, let's do it in there rather than blocking this PR.

Reply:
@shahi645 I think we should either not support average checks or we need to provide descriptive validation messages for when those checks fail. Otherwise this is a bug report waiting to happen.

Reply:
@nealedj I can add the a average error messages, but the content needs to be approved and supplied

Reply:
@shahi645 just don't support them, We have the ability to easily add them, but they are not a requirement so get rid for now to make this PR simpler